components/flow-orchestration-framework/org.wso2.carbon.identity.flow.mgt/src/main/java/org/wso2/carbon/identity/flow/mgt/model/FlowDTO.java

```diff
@@ -18,14 +18,16 @@
 
 package org.wso2.carbon.identity.flow.mgt.model;
 
+import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.List;
 
 /**
  * DTO class for Flow.
  */
-public class FlowDTO {
+public class FlowDTO implements Serializable {
 
+    private static final long serialVersionUID = 1L;
```

Comment:
Use upper case for constant

Reply:
IINM we can't change this variable as this is being consumed by java.io.Serializable.